src/conn/conn_open.c

```diff
@@ -210,9 +210,18 @@ __wt_connection_workers(WT_SESSION_IMPL *session, const char *cfg[])
     /*
      * Run recovery. NOTE: This call will start (and stop) eviction if recovery is required.
      * Recovery must run before the history store table is created (because recovery will update the
-     * metadata), and before eviction is started for real.
+     * metadata, and set the maximum file id seen), and before eviction is started for real.
```

Comment:
I'm surprised this change is here - was it a merge conflict issue?

Reply:
Yeah. The merge is a mess.